app/models/ucas_match.rb

```diff
@@ -91,6 +101,16 @@ def application_choices_for_same_course_on_both_services
     ucas_matched_applications.select(&:both_scheme?).map(&:application_choice)
   end
 
+  def ucas_withdrawal?
+    ucas_matched_applications.select(&:both_scheme?) &&
+      ucas_matched_applications.select(&:application_withdrawn_on_ucas?).any?
```

Comment:
I don’t think it explicitly checks if the duplicate application was withdrawn from UCAS. The candidate could have withdrawn from some other course they applied for on UCAS. 

I think it would return `true` for the following scenario:
<img width="669" alt="Screenshot 2020-12-02 at 12 11 28" src="https://user-images.githubusercontent.com/38078064/100872902-6d66e580-349a-11eb-994c-79443dbad9f5.png">

Maybe we could also call the method something more descriptive.

Reply:
```suggestion
      ucas_matched_applications.select(&:both_scheme?).select(&:application_withdrawn_on_ucas?).any?      
```


@AgaDufrat if I understand correctly should the condition then be updated to only check if the applications that are on both schemes have been withdrawn similar to my suggestions and rename it to something like `duplicate_application_withdrawn_from_ucas?`

Reply:
Yes, that's what I was thinking and I like the name!

You may also want to consider dealing with more than one duplicate application choices withdrawn from UCAS. 
I described my idea here (we probably wouldn't need public `duplicate_application_withdrawn_from_ucas?`): https://github.com/DFE-Digital/apply-for-teacher-training/pull/3552/files#r534232874